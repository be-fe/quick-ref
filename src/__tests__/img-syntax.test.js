/**
 * @file img-syntax.test
 * @author Cuttle Cong
 * @date 2018/5/21
 * @description
 */
const nps = require('path')

const compile = require('../transpiler/index')

describe('img-syntax', function() {
  it('should img-syntax normal', done => {
    compile(
      nps.join(__dirname, 'fixture/img-syntax.md'),
      {
        imgAlias: {
          avalon: nps.join(__dirname, 'fixture/avalon.png')
        }
      },
      ['img']
    )
      .then(md => expect(md).toMatchSnapshot())
      .then(done)
  })
})
