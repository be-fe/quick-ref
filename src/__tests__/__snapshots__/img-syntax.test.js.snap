// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`img-syntax should img-syntax normal 1`] = `
Object {
  "input": "@img:[\`fs\`]avalon#ssd@hahahah
@img:avalon说说@hahahah
@img:aval是sswzww@hahahah

\`\`\`markdown
@img:[\`fs\`]avalon#ssd@hahahah
@img:avalon说说@hahahah
@img:aval是sswzww@hahahah
\`\`\`
",
  "output": "![\\\\\`fs\\\\\`](avalon.png#ssd)hahahah
@img:avalon说说@hahahah
@img:aval是sswzww@hahahah

\`\`\`markdown
@img:[\`fs\`]avalon#ssd@hahahah
@img:avalon说说@hahahah
@img:aval是sswzww@hahahah
\`\`\`
",
}
`;
